sample_install_only_java_env.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# The LabKey Install-Script is broken out into to step functions.  Each STEP can be skipped by exporting the
```

Comment:
broken out into to step > broken out into step

Reply:
edited